### ISSUE TITLE
Use HTML form and DOM object FormData with $.ajax for upload

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/rails-paperclip-s3-08#1]
Rather than rails views.

use `name="root[property]"` on form inputs to get strong params.

**Comment by GA-MEB** [comment]
> use name="root[property]" on form inputs to get strong params.

I just hard-coded it:

``` javascript
data: { movie: {
  title: $('#title').val(),
  release_year: $('#release').val(),
  poster: event.target.result
} }
```

**Comment by gaand** [comment]
You need to use FormData to handle this cleanly.

**Comment by GA-MEB** [comment]
Can you please give an example implementation? My imagined setup with FormData is below, but I'm not sure that it's correct.

``` javascript
var formData = new FormData(/* form */);
```

and then

``` javascript
$.ajax({
    //...
   data: { "movie": formData}
})
```

**Comment by GA-MEB** [comment]
Takeaway from discussion wth @gaand : Consider making a mini-repo on FormData, and introducing it in Unit 1 in the context of the register and login forms.

**Comment by MattB-GA-Boston** [comment]
Added this to `meta`

https://github.com/ga-wdi-boston/meta/issues/24

**Comment by gaand** [comment]
I don't believe this issue should be closed since this issue is open:  https://github.com/ga-wdi-boston/rails-paperclip-s3/issues/9

**Comment by gaand** [comment]
Probably addressed by #14
